ckanext/recombinant/views.py

```diff
@@ -211,39 +227,42 @@ def _xlsx_response_headers():
     user_agent_legacy = getattr(request, 'headers', {}).get('User-Agent')
     user_agent = getattr(request, 'headers', {}).get('Sec-CH-UA', user_agent_legacy)
     if user_agent and (
-        "Microsoft Edge" in user_agent or
-        "Edg/" in user_agent or
-        "EdgA/" in user_agent
-        ):
-            # force the XLSX file to be downloaded in MS Edge,
-            # and not open in Office Apps Online.
-            content_type = 'application/octet-stream'
-            disposition_type = 'attachment'
+      "Microsoft Edge" in user_agent or
+      "Edg/" in user_agent or
+      "EdgA/" in user_agent):
+        # force the XLSX file to be downloaded in MS Edge,
+        # and not open in Office Apps Online.
+        content_type = 'application/octet-stream'
+        disposition_type = 'attachment'
```

Comment:
switching to 2-space indents?

Reply:
only looks like 2 spaces, flake8 was being annoying with the if statement. So the if is lined up with the opening bracket of it, and then the following statement is still 4 spaces from the if. But then it just looks like it is 2 spaces from the bracket wrapped if statement.

Reply:
oh I see. Usually I'll add an extra indent (go to 8 spaces) when an if condition wraps to the next line to differentiate it from the block inside (4 spaces). Interesting that flake8 is happy with having it floating in between at 2 spaces, I've never seen that.